tests/test_ezsp.py

```diff
@@ -189,3 +189,25 @@ def test_callback_exc(ezsp_f):
     ezsp_f.add_callback(testcb)
     ezsp_f.handle_callback(1)
     assert testcb.call_count == 1
+
+
+def test_version_5(ezsp_f):
+    loop = asyncio.get_event_loop()
+    ezsp_f._gw = mock.MagicMock()
+
+    ezsp_f.frame_received(b'\x00\x00\xff\x00\x00\x05\x05\x06')
+    assert ezsp_f.ezsp_version == 5
+
+    ezsp_f.getValue(1)
+    assert ezsp_f._gw.data.call_count == 1
+
+    @asyncio.coroutine
```

Comment:
_command is not a coroutine, so the thing mocking it should probably not be a coroutine.